.github/workflows/markdown.yaml

```diff
@@ -19,77 +19,79 @@ on:
     inputs:
       config-links-path:
         required: false
-        default: '.github/config/md-link-config.json'
+        default: ".github/config/md-link-config.json"
         type: string
       config-lint-path:
-        default: '.github/config/.markdownlint.json'
+        default: ".github/config/.markdownlint.json"
         required: false
         type: string
 jobs:
   markdown:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
-      with:
-        token: ${{ secrets.REPO_PAT }}
-        ref: ${{github.event.pull_request.head.ref}}
-        repository: ${{github.event.pull_request.head.repo.full_name}}
-    - name: Prettify MD files
-      uses: zaphiro-technologies/prettier_action@main
-      with:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.REPO_PAT }}
+          ref: ${{github.event.pull_request.head.ref}}
+          repository: ${{github.event.pull_request.head.repo.full_name}}
+      - name: Remove vendor directory
+        run: |
+          rm -rf vendor
```

Comment:
This should do the trick for now @chicco785

Reply:
let's give a try

Reply:
issue is with line 65, let's move your command after the commit of the linter or it will commit the deletion of your vendor folder